__tests__/commands/add.js

```diff
@@ -986,3 +986,42 @@ test.concurrent('preserves unaffected bin links after adding to workspace packag
     expect(await fs.exists(`${config.cwd}/packages/workspace-2/node_modules/.bin/workspace-1`)).toEqual(true);
   });
 });
+
+test.concurrent('installs "latest" instead of maxSatisfying if it satisfies requested pattern', (): Promise<void> => {
+  /* eslint-disable */
```

Comment:
Can you just disable the relevant warning with `/* eslint-disable rule-name */` instead of a blanket disable statement in case the enable part goes away?

Or ideally, we wouldn't need this at all since you can use https://git.io/ to get short GitHub URLs ;)